 (line: 4) - APPLICATION: [Feb/02/15:23:09:23] - Request was brought back with error => {"category":"authorization","code":"access-denied","message":"The authenticated user does not have sufficient permissions to access this resource","details":[{"name":"requestMethod","value":"GET"},{"name":"remoteAddress","value":"127.0.0.1"},{"name":"requestPath","value":"\/users\/__USER_ID__\/phoneNumbers"}]}
 (line: 4) - APPLICATION: [Feb/02/15:23:09:25] - Request was brought back with error => {"category":"authorization","code":"access-denied","message":"The authenticated user does not have sufficient permissions to access this resource","details":[{"name":"requestMethod","value":"GET"},{"name":"remoteAddress","value":"127.0.0.1"},{"name":"requestPath","value":"\/users\/__USER_ID__\/phoneNumbers"}]}
 (line: 4) - APPLICATION: [Feb/02/15:23:09:25] - Request was brought back with error => {"category":"authorization","code":"access-denied","message":"The authenticated user does not have sufficient permissions to access this resource","details":[{"name":"requestMethod","value":"GET"},{"name":"remoteAddress","value":"127.0.0.1"},{"name":"requestPath","value":"\/users\/__USER_ID__\/phoneNumbers"}]}
 (line: 4) - APPLICATION: [Feb/02/15:23:13:36] - Request was brought back with error => {"category":"authorization","code":"access-denied","message":"The authenticated user does not have sufficient permissions to access this resource","details":[{"name":"requestMethod","value":"GET"},{"name":"remoteAddress","value":"127.0.0.1"},{"name":"requestPath","value":"\/users\/__USER_ID__\/phoneNumbers"}]}
